### PR TITLE
:bug: Handles browsers without console

### PR DESCRIPTION
**Pull request by lboyette-okta**
Resolves: OKTA-115460

@rchild-okta 
@jhe-okta

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
...
-}
+};
+
+util.getNativeConsole = function() {
+  return window.console;
+};
 
-function warn(text) {
+util.getConsole = function() {
+  var nativeConsole = util.getNativeConsole();
```
Comment (rchild-okta) [inline lib/util.js]: Double checking - this doesn't throw an error does it? I'm guessing it does not, but we did have issues in the plugin (and I'm mostly guessing the issues in the plugin were related to a long time ago issue where we didn't have access to the window object)

File: lib/util.js [hunk 2]
```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
...
+  if (nativeConsole && nativeConsole.log) {
+    return window.console;
```
Comment (rchild-okta) [inline lib/util.js]: Shouldn't we just return nativeConsole here?

File: lib/util.js [hunk 3]
```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
   } else {
     return link;
   }
-}
+};
+
+util.getNativeConsole = function() {
```
Comment (rchild-okta) [inline lib/util.js]: Am curious how often we call console (guessing not often) - it's probably okay this way, but might be nicer to move this to a singleton type pattern where we're only making this check once.

Comment (lboyette-okta) [inline lib/util.js]: this makes it possible to test

File: lib/util.js (hunk 1 above)
Comment (lboyette-okta) [inline lib/util.js]: I'll make sure this works for @nikhilvenkatraman-okta before merging. I don't think it throws an error, I just don't think console exists.

File: lib/util.js (hunk 2 above)
Comment (lboyette-okta) [inline lib/util.js]: smh, yes

File: lib/util.js (hunk 1 above)
Comment (lboyette-okta) [inline lib/util.js]: Tested with @nikhilvenkatraman-okta. In the tested app, console doesn't exist, so these changes work.

**Pull request merged**